generate_flocker_certs.sh

```diff
@@ -270,11 +337,10 @@ generate_and_sign_cert "$output_dir/control-service" \
 echo
 
 echo "Generating node keypair(s)"
-for node_hostname in ${nodes[@]}; do
-  generate_and_sign_cert "$output_dir/node-$node_hostname" \
-                         "/CN=node-$(uuidgen)/OU=$cluster_uuid" \
-                         "$cluster_keypair_path"
-done
+# XXX: Array passed as a name, not a value
+generate_and_sign_node_certs $cluster_keypair_path \
+                             "nodes[@]" \
+                             $output_dir
 echo
 
 echo "Generating API keypair"
```

Comment:
not sure I like that we always name this "plugin" but this was introduced elsewhere and we can fix it later.